CHANGELOG.md

```diff
@@ -1,25 +1,43 @@
-# Unreleased
+# Unreleased: 12.0.0 (Breaking release)
 
-## Features
+## Breaking changes 
 
-### Preserve query strings
+This release ensures the GOV.UK Prototype Kit reflects the latest release of the GOV.UK Frontend, v4.0.0.
 
-We now preserve URL query strings when redirecting POST requests to GET requests.
+### Update to GOV.UK Frontend v4.0.0 [(#1195)](https://github.com/alphagov/govuk-prototype-kit/pull/1195)
 
-This means if you have a query like `/link/to/something?query=true&hello=world` on your POST form action, and you submit the form, the URL query string will be present in the redirected URL.
+In this release:
```

Comment:
```suggestion
The new release of GOV.UK Frontend contains:
```
Reason for suggestion: to differentiate release of GOV.UK Frontend from "this release" (i.e., of the Kit) mentioned on line 5.